jenkins-remoting

```diff
@@ -0,0 +1,52 @@
+package:
+  name: jenkings-remoting
+  version: 4.14
```

Comment:
```suggestion
  version: 3206.vb_15dcf73f6a_9
```

The releases all seem to follow this naming convention. I can't figure out what it means though! https://github.com/jenkinsci/remoting/releases